assets/vue/components/BlockedIncident.vue

```diff
@@ -51,16 +52,30 @@ export default {
         const {version} = value.linkinfo;
         const {groupid} = value.linkinfo;
         const newkey = `${groupid}:${version}`;
-        if (!(newkey in results)) {
-          results[newkey] = {name: value.name, passed: 0, failed: 0, stopped: 0, waiting: 0};
-          results[newkey].linkinfo = value.linkinfo;
-          results[newkey].linkinfo.flavor = [];
+        if (this.groupNames.toLowerCase().split(',').includes(value.name.toLowerCase()) || this.groupNames == '') {
+          if (!(newkey in results)) {
+            results[newkey] = {name: value.name, passed: 0, failed: 0, stopped: 0, waiting: 0};
+            results[newkey].linkinfo = value.linkinfo;
+            results[newkey].linkinfo.flavor = [];
+          }
+          results[newkey].linkinfo.flavor.push(flavor);
+          results[newkey].passed += value.passed || 0;
+          results[newkey].stopped += value.stopped || 0;
+          results[newkey].waiting += value.waiting || 0;
+          results[newkey].failed += value.failed || 0;
+        }
+      }
+      return results;
+    },
+    incidentResultsGrouped() {
+      if (this.groupNames == '') {
+        return this.incidentResults;
+      }
+      const results = [];
+      for (const value of Object.values(this.incidentResults)) {
+        if (this.groupNames.toLowerCase().split(',').includes(value.name.toLowerCase())) {
```

Comment:
Do we really need to compute `this.groupNames.toLowerCase().split(',')` again in every loop iteration?